exporter/awscloudwatchlogsexporter/factory.go

```diff
@@ -55,6 +55,8 @@ func createLogsExporter(_ context.Context, set component.ExporterCreateSettings,
 			Enabled:      true,
 			NumConsumers: 1, // due to the sequence token, there can be only one request in flight
 		}),
+		exporterhelper.WithStart(exporter.Start),
+		exporterhelper.WithShutdown(exporter.Shutdown),
```

Comment:
Very nit: move this as first 2 options :)

Reply:
Sure, done.